avm/res/databricks/workspace/main.bicep

```diff
@@ -96,9 +96,23 @@ param publicNetworkAccess string = 'Enabled'
 ])
 param requiredNsgRules string = 'AllRules'
 
+@description('Optional. Determines wether the managed storage account should be private or public. For security reasons, it is recommended Enabled this.')
```

Comment:
Please change to "Determines whether the managed storage account should be private or public. For best security practices, it is recommended to set it to Enabled"